sechub-solution/docker/SecHub-Debian.dockerfile

```diff
@@ -163,6 +163,7 @@ RUN groupadd --gid "$GID" "$USER" && \
     useradd --uid "$UID" --gid "$GID" --no-log-init --create-home "$USER"
 
 RUN mkdir --parent "$SECHUB_FOLDER/secrets" "$SECHUB_STORAGE_SHAREDVOLUME_UPLOAD_DIR"
+# Mounted secret files (like e.g. SSL certificates) go to $SECHUB_FOLDER/secrets. See deployment.yaml file.
```

Comment:
this comment is in the wrong line: `COPY --from=builder "$SECHUB_ARTIFACT_FOLDER" "$SECHUB_FOLDER"` does not do anything with secrets.